docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,185 @@ tags:
+In this guide, we'll demonstrate how to use [Jans Tarp](../../demos/jans-tarp) to register a client on an authorization server, and use the embedded cedarling webassembly app to reach an authorization decision. This will be done in three steps:
 
-## Have questions in the meantime?
+1. [Setting up a policy store](#policy-store-setup)
+2. [Loading the policy store in Jans Tarp](#tarp-setup)
+3. [Using Tarp to launch an authorization flow and validate authorization against the policy](#authentication-flow)
 
-While this documentation is in progress, you can ask questions through [GitHub Discussions](https://github.com/JanssenProject/jans/discussions) or the [community chat on Gitter](https://gitter.im/JanssenProject/Lobby). Any questions you have will help determine what information our documentation should cover.
 
-## Want to contribute?
+## Prerequisites
 
-If you have content you'd like to contribute to this page in the meantime, you can get started with our [Contribution guide](https://docs.jans.io/head/CONTRIBUTING/).
+Before we begin, we need to meet the following requirements:
+
+* [Firefox](https://www.mozilla.org/en-US/firefox/windows/) or [Google Chrome](https://www.google.com/chrome/index.html)
+* The latest release of [Jans Tarp](https://github.com/JanssenProject/jans/releases/tag/nightly). Download the zip file for your browser and install it.
+
+## Cedar Policy
+
+[Cedar](https://www.cedarpolicy.com/en) is a language for defining permissions as policies, and a specification for evaluating those policies. Cedarling expects policies to be written in Cedar. For this quickstart, we will use a policy where access is only granted if the authenticated user has the `SupremeRuler` role. In cedar, this is expressed like so:
+
+```
+@id("allow_supreme_ruler")
+permit(
+  principal is Jans::User,
+  action,
+  resource
+)
+when {
+  principal has userinfo_token.role &&
+  principal.userinfo_token.role.contains("SupremeRuler")
+};
+```
+
+This policy expects that the authorized entity is of type `Jans::User` (as defined in the [default Janssen schema](../../jans-cedarling/schema/cedarling-core.cedarschema)). Then the policy tests whether the userinfo token obtained after authorization contains the `role` claim as well as whether the claim contains the value `SupremeRuler`. If all these conditions are met, Cedarling will allow access. This is also called Role Based Access Control (RBAC).
+
+Further information on cedar can be found in the [official documentation](https://docs.cedarpolicy.com/)
+
+## Policy Store Setup
+
+To begin using Cedarling, we need to set up a policy store. We will use [Agama Lab](https://cloud.gluu.org/agama-lab) for this. Please follow this video guide to set up your policy store:
+
+![agama-lab-policy-store](../assets/agama-lab-policy-store.mp4)
+
+For the trusted issuer token metadata, paste in the following content:
```

Comment:
This looks pretty standard... is this the default?

Reply:
No, the default input is an example that needs to be modified for our use case.